monitoring/osd-configuring-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,117 @@
+
+[id="prerequisites"]
+== Prerequisites
+
+* The monitoring stack imposes additional resource requirements. Consult the computing resources recommendations in link:https://docs.openshift.com/container-platform/4.6/scalability_and_performance/scaling-cluster-monitoring-operator.html[Scaling the Cluster Monitoring Operator] and verify that you have sufficient resources.
+
+// Maintenance and support for monitoring
+include::modules/osd-monitoring-maintenance-and-support.adoc[leveloffset=+1]
+include::modules/osd-monitoring-support-considerations.adoc[leveloffset=+2]
+include::modules/osd-monitoring-unmanaged-monitoring-operators.adoc[leveloffset=+2]
+
+// Configuring the monitoring stack
+include::modules/osd-monitoring-configuring-the-monitoring-stack.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+
+// Configurable monitoring components
+include::modules/osd-monitoring-configurable-monitoring-components.adoc[leveloffset=+1]
+
+// Moving monitoring components to different nodes
+include::modules/osd-monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.6/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes]
+* link:https://docs.openshift.com/container-platform/4.6/nodes/scheduling/nodes-scheduler-node-selectors.html[Placing pods on specific nodes using node selectors]
+* See the link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector[Kubernetes documentation] for details on the `nodeSelector` constraint
+
+// Assigning tolerations to monitoring components
+include::modules/osd-monitoring-assigning-tolerations-to-monitoring-components.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../monitoring/osd-configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
+* xref:../monitoring/osd-enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
```

Comment:
I think this is causing the build error. It might not be relevant to ROSA if that is enabled by default, but please check because my ROSA knowledge is limited. Thanks!

Reply:
its enabled by default for both ROSA and OSD.

Reply:
It is not relevant, so I removed it. Thanks!